### PR TITLE
MCP-2127 Checkstyle fixes for app module

### PR DESCRIPTION
**Pull request by ianwolson**
## What was the problem?
We had around 50 checkstyle warnings in the app module which needed to be fixed.

Associated tickets or Slack threads:

- [MCP-2127](https://amida.atlassian.net/browse/MCP-2127)

## How does this fix it?[^1]
Gets rid of checkstyle warnings in the build logs.

## How to test this PR
- Step 1 build project
- Step 2 verify that in the logs there are no checkstyle warnings for the app module

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Comment by ianwolson** [comment]
I couldn't get rid of the lambda indentations issues because spotless apply reverts the indents back to where they were after addressing them.

File: app/src/test/java/gov/va/vro/config/MasAuthTest.java [hunk 1]
```diff
@@ -34,29 +37,43 @@ public class MasAuthTest extends BaseControllerTest {
   @Test
   void testAuthenticateHeader() {
     assertFalse(apiAuthKeyManagerMock.authenticate(authentication).isAuthenticated());
-    String sampleJWT =
-        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImMwOTI5NTJlLTM4ZDYtNDNjNi05MzBlLWZmOTNiYTUxYjA4ZiJ9.eyJleHAiOjk5OTk5OTk5OTksImlhdCI6MTY0MTA2Nzk0OSwianRpIjoiNzEwOTAyMGEtMzlkOS00MWE4LThlNzgtNTllZjAwYTlkNDJlIiwiaXNzIjoiaHR0cHM6Ly9zYW5kYm94LWFwaS52YS5nb3YvaW50ZXJuYWwvYXV0aC92Mi92YWxpZGF0aW9uIiwiYXVkIjoibWFzX2RldiIsInN1YiI6IjhjNDkyY2NmLTk0OGYtNDQ1Zi05NmY4LTMxZTdmODU5MDlkMiIsInR5cCI6IkJlYXJlciIsImF6cCI6Im1hc19kZXYiLCJzY29wZSI6Im9wZW5pZCB2cm9fbWFzIiwiY2xpZW50SWQiOiJtYXNfZGV2In0.Qb41CR1JIGGRlryi-XVtqyeNW73cU1YeBVqs9Bps3TA";
+    String sampleJwt =
```
Comment (dimitri-amida) [inline app/src/test/java/gov/va/vro/config/MasAuthTest.java:40]: We need to copy and paste this, so this format does not work very well. You can use Java 17 multiline string: https://www.baeldung.com/java-multiline-string

File: app/src/test/java/gov/va/vro/config/MasAuthTest.java [hunk 2]
```diff
@@ -34,29 +37,43 @@ public class MasAuthTest extends BaseControllerTest {
...
     assertTrue(apiAuthKeyManagerMock.authenticate(authentication).isAuthenticated());
   }
 
   @Test
-  void testAuthenticateLhAPI() {
+  void testAuthenticateLhApi() {
     assertFalse(apiAuthKeyManagerMock.authenticate(authentication).isAuthenticated());
-    String sampleJWT =
-        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImMwOTI5NTJlLTM4ZDYtNDNjNi05MzBlLWZmOTNiYTUxYjA4ZiJ9.eyJleHAiOjk5OTk5OTk5OTksImlhdCI6MTY0MTA2Nzk0OSwianRpIjoiNzEwOTAyMGEtMzlkOS00MWE4LThlNzgtNTllZjAwYTlkNDJlIiwiaXNzIjoiaHR0cHM6Ly9zYW5kYm94LWFwaS52YS5nb3YvaW50ZXJuYWwvYXV0aC92Mi92YWxpZGF0aW9uIiwiYXVkIjoibWFzX2RldiIsInN1YiI6IjhjNDkyY2NmLTk0OGYtNDQ1Zi05NmY4LTMxZTdmODU5MDlkMiIsInR5cCI6IkJlYXJlciIsImF6cCI6Im1hc19kZXYiLCJzY29wZSI6Im9wZW5pZCB2cm9fbWFzIiwiY2xpZW50SWQiOiJtYXNfZGV2In0.Qb41CR1JIGGRlryi-XVtqyeNW73cU1YeBVqs9Bps3TA";
+    String sampleJwt =
```
Comment (dimitri-amida) [inline app/src/test/java/gov/va/vro/config/MasAuthTest.java:59]: same as previous comment

File: app/src/test/java/gov/va/vro/controller/BaseControllerTest.java [hunk 3]
```diff
@@ -56,6 +56,13 @@ protected <I, O> ResponseEntity<O> exchange(
     return exchange(url, request, method, null, responseType);
   }
 
-  String sampleJWT =
-      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImMwOTI5NTJlLTM4ZDYtNDNjNi05MzBlLWZmOTNiYTUxYjA4ZiJ9.eyJleHAiOjk5OTk5OTk5OTksImlhdCI6MTY0MTA2Nzk0OSwianRpIjoiNzEwOTAyMGEtMzlkOS00MWE4LThlNzgtNTllZjAwYTlkNDJlIiwiaXNzIjoiaHR0cHM6Ly9zYW5kYm94LWFwaS52YS5nb3YvaW50ZXJuYWwvYXV0aC92Mi92YWxpZGF0aW9uIiwiYXVkIjoibWFzX2RldiIsInN1YiI6IjhjNDkyY2NmLTk0OGYtNDQ1Zi05NmY4LTMxZTdmODU5MDlkMiIsInR5cCI6IkJlYXJlciIsImF6cCI6Im1hc19kZXYiLCJzY29wZSI6Im9wZW5pZCB2cm9fbWFzIiwiY2xpZW50SWQiOiJtYXNfZGV2In0.Qb41CR1JIGGRlryi-XVtqyeNW73cU1YeBVqs9Bps3TA";
+  String sampleJwt =
```
Comment (dimitri-amida) [inline app/src/test/java/gov/va/vro/controller/BaseControllerTest.java:59]: Also please multi-line string here

Comment (ianwolson) [reply app/src/test/java/gov/va/vro/controller/BaseControllerTest.java:59]: Okay no problem, I just tried the first method at that link and for some reason the test fails.  I will look into this further.

**Comment by ianwolson** [comment]
@dimitri-amida 
The problem with the text block was that the new line needs to be ignored within using \
Unfortunately this will still mess up your copy paste, but at least it's not as bad as the concatenation clean-up wise.  Does this seem okay, or should I look for another solution?

**Review by dimitri-amida — APPROVED** [review]
👍

File: app/src/main/java/gov/va/vro/config/MasOauth2Config.java [hunk 4]
```diff
@@ -35,23 +35,23 @@ ClientRegistration masAuthProviderClientRegistration(
...
       ClientRegistrationRepository clientRegistrationRepository) {
     return new InMemoryOAuth2AuthorizedClientService(clientRegistrationRepository);
   }
 
   /**
-   * Create the authorized client manager and service manager using the beans created and configured
-   * above
+   * Create the authorized client manager and service manager using the beans created and
+   * configured. above
    */
```
Comment (au2) [inline app/src/main/java/gov/va/vro/config/MasOauth2Config.java:55]: Looks like "." is in the wrong place.

**Review by au2 — APPROVED** [review]
Put a minor comment. You can fix in the next round.

**Pull request merged**